File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    let answer = 0;
    for (let i = 0; i <= t.length - p.length; i++) {
        let num = t.substr(i, p.length)
        if (Number(num) <= Number(p)) {
            answer++
        }
    }
    return answer;
}

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    answer = 0
    p_len = len(p)
    p_int = int(p)
    for i in range(len(t) - p_len + 1):
        sub_str = t[i : i + p_len]
        if int(sub_str) <= p_int:
            answer += 1
    return answer